README.md

```diff
@@ -8,8 +8,11 @@ at the heart of the [Wikibase software](http://wikiba.se/).
 
 ## Release notes
 
-### 3.0.0 (alpha)
+### 3.0.0 (2016-08-02)
 
+* Added `Set::toArray`.
+* `Fingerprint::setLabel`, `setDescription` and `setAliases` remove the item when null or an empty
```

Comment:
I suggest replacing "item" with something else here as it confoosed me for a few seconds. Perhaps term?